Setup.bat

```diff
@@ -60,9 +60,10 @@ call :MarkStartOfBlock "Setup variables"
     set BINARIES_DIR=%~dp0SpatialGDK\Binaries\ThirdParty\Improbable
 
     rem Copy schema to the projects spatial directory.
-    set SCHEMA_COPY_DIR=%~dp0..\..\..\spatial\schema\unreal\gdk
-    set SCHEMA_STD_COPY_DIR=%~dp0..\..\..\spatial\build\dependencies\schema\standard_library
     set SPATIAL_DIR=%~dp0..\..\..\spatial
+    set SPATIAL_TOOLS_COPY_DIR=%SPATIAL_DIR%\tools

```

Comment:
Is this necessary over just using them in their default extracted location?

Reply:
strictly, no, although it's much more convenient.

Reply:
compare `spatial\tools\snapshot_converter` to `Game\Plugins\UnrealGDK\SpatialGDK\Binaries\ThirdParty\Improbable\Programs\snapshot_converter`

Reply:
Yep fair enough!